tests/test_cli.py

```diff
@@ -440,23 +437,6 @@ async def test_run(self):
         )
 
 
-class TestAccuracy(RecordsTestCase):
```

Comment:
Why are we removing this?

Reply:
I will be adding the fakescorer here and fixing this test :)